src/index.js

```diff
@@ -1,7 +1,21 @@
 const React = require('react');
 
-// Create random ID
-const randomID = () => `${(Number(String(Math.random()).slice(2)) + Date.now() + Math.round(performance.now())).toString(36)}`;
+// Borrowed from https://github.com/ai/nanoid/blob/3.0.2/non-secure/index.js
+// This alphabet uses `A-Za-z0-9_-` symbols. A genetic algorithm helped
+// optimize the gzip compression for this alphabet.
+let urlAlphabet =
+  'ModuleSymbhasOwnPr-0123456789ABCDEFGHNRVfgctiUvz_KqYTJkLxpZXIjQW'
+
+export let nanoid = (size = 21) => {
```

Comment:
Since this function and the above variable do not change, could you make them`const` instead of `let`.

Reply:
All set.